src/Controller/Component/ModulesComponent.php

```diff
@@ -62,6 +62,17 @@ class ModulesComponent extends Component
      */
     protected $modules = [];
 
+    /**
+     * Other "logic" modules, non objects
+     *
+     * @var array
+     */
+    protected $otherModules = [
```

Comment:
Suggested change:

1. add `tags` module with color in default `config/app.php` 
2. in `$otherModules` list only module names, like 
```php 
protected $otherModules = [
    'tags', 
];
```
3. in `getModules()` change  
 ```$metaModules = $this->modulesFromMeta();```
to 
 ``` $metaModules = $this->modulesFromMeta() + array_flip($this->otherModules);```
4. remove `$this->otherModules` from `array_merge` below

Reply:
ok. but this way the color is not preserved, because `Modules` config from `config/app.php` is almost always overwritten